src/include/daos/mem.h

```diff
@@ -632,4 +632,132 @@ umem_tx_add_callback(struct umem_instance *umm, struct umem_tx_stage_data *txd,
 	D_ASSERT(umm->umm_ops->mo_tx_add_callback != NULL);
 	return umm->umm_ops->mo_tx_add_callback(umm, txd, stage, cb, data);
 }
+
+/*********************************************************************************/
+
+typedef uint64_t	daos_addr_t;
+
+/** Describing a storage region for I/O */
+struct umem_store_region {
+	/** start offset of the region */
+	daos_addr_t	sr_addr;
+	/** size of the region */
+	daos_size_t	sr_size;
+};
+
+/** I/O descripter, it can include arbitrary number of storage regions */
```

Comment:
I think spelling issues will cause issues for subsequent commits, please fix it.

Reply:
OK, thanks